cpp/src/command_classes/CommandClass.cpp

```diff
@@ -329,12 +329,15 @@ void CommandClass::ReadValueRefreshXML
 	char const* str;
 	bool ok = false;
 	const char *genre;
+	int32 intVal;
```

Comment:
QueryIntAttribute expects an int32 type, as you do in the rest of the code (same method)
You parse a uint8 pointer, and this caused a crash